src/components/m-display-value.vue

```diff
@@ -0,0 +1,87 @@
+
+<template>
+  <p :class="numberColorClass">{{ valueDisplayed }}</p>
+</template>
+
+<script>
+import { roundNumber } from "@/utils/number";
+
+type SignType = "euro" | "percent" | "point" | "null"
```

Comment:
J'aurais plutôt parlé de `currency` parce qu'on peut avoir des marques qui ne bossent pas en euro.

```suggestion
type SignType = "currency" | "percent" | "point" | "null"
```